db/config.cc

```diff
@@ -411,56 +411,52 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     */
     , commit_failure_policy(this, "commit_failure_policy", value_status::Unused, "stop",
         "Policy for commit disk failures:\n"
-        "\n"
-        "\tdie          Shut down gossip and Thrift and kill the JVM, so the node can be replaced.\n"
-        "\tstop         Shut down gossip and Thrift, leaving the node effectively dead, but can be inspected using JMX.\n"
-        "\tstop_commit  Shut down the commit log, letting writes collect but continuing to service reads (as in pre-2.0.5 Cassandra).\n"
-        "\tignore       Ignore fatal errors and let the batches fail."
+        "* die          Shut down gossip and Thrift and kill the JVM, so the node can be replaced.\n"
+        "* stop         Shut down gossip and Thrift, leaving the node effectively dead, but can be inspected using JMX.\n"
+        "* stop_commit  Shut down the commit log, letting writes collect but continuing to service reads (as in pre-2.0.5 Cassandra).\n"
+        "* ignore       Ignore fatal errors and let the batches fail."
```

Comment:
Note the help text is also output when you run `scylla --help`. I guess we can adjust it later to get better output.

Reply:
Good catch, I wasn't aware of that. To ensure everything looks alright, I'll try to build ScyllaDB and run the `--help`  command to check how it looks like and make any needed adjustments before merging.

Reply:
i applied the change, and the output of `scylla -help` can be found at https://gist.github.com/tchaikov/f56bae6920a90e4d9f9de515689f1378 . because the size of the output, i could not paste it as a comment here.